Automattic/pocket-casts-ios
# Starred: display starred on Profile for logged out users

Fixes #681

Show starred for logged out users

## To test

1. Do a clean install of the app
2. Star some episodes
3. Go to Profile
4. Tap Starred
5. ✅ The episodes you starred should display
6. Delete and reinstall the app
7. Login with an account
8. Go to Profile
9. Tap Starred
10. ✅ The episodes you starred should display

## Checklist
